empatia.tf

```diff
@@ -0,0 +1,141 @@
+resource "aws_iam_user" "empatia" {
+  name = "empatia"
+}
+
+resource "aws_iam_access_key" "empatia" {
+  user = aws_iam_user.empatia.name
+}
+
+resource "aws_route53_zone" "empatia" {
+  name = "bankempatii.pl"
+}
+
+resource "aws_route53_record" "a_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = aws_route53_zone.empatia.name
+  type    = "A"
+  alias {
+    evaluate_target_health = false
+    name                   = "d6cxmr8432pje.cloudfront.net"
+    zone_id                = "Z2FDTNDATAQYW2"
+  }
+}
+
+resource "aws_route53_record" "a_www_empatia" {
```

Comment:
That should probably redirect to `bankempatii.pl`. Currently, this record points to the nonexisting resource in OVH. We should remove it.